<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>league_editor_dialog</class>
 <widget class="QDialog" name="league_editor_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>League Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="league_label">
       <property name="text">
        <string>Finalize this League?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="league_name_label">
       <property name="text">
        <string>League's Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="league_name_line_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="import_button">
       <property name="text">
        <string>Import League</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_button">
       <property name="text">
        <string>Export League</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="league_editor_list_widget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="delete_team_button">
         <property name="text">
          <string>Delete Team</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="edit_team_button">
         <property name="text">
          <string>Edit Team</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="new_team_label">
       <property name="text">
        <string>Team's name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="team_name_line_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="add_team_button">
       <property name="text">
        <string>Add Team</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>league_name_line_edit</tabstop>
  <tabstop>team_name_line_edit</tabstop>
  <tabstop>add_team_button</tabstop>
  <tabstop>edit_team_button</tabstop>
  <tabstop>delete_team_button</tabstop>
  <tabstop>import_button</tabstop>
  <tabstop>export_button</tabstop>
  <tabstop>league_editor_list_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>league_editor_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>league_editor_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
